src/mouse/default_gutter_handler.js

```diff
@@ -133,6 +133,10 @@ class GutterTooltip extends Tooltip {
     constructor(editor) {
         super(editor.container);
         this.editor = editor;
+        /**
+        * @type Number
```

Comment:
I think this should be `@type {number | undefined}` (curly brackets around the type)? 

(see the examples here https://jsdoc.app/tags-type)